src/precess_cd.jl

```diff
@@ -0,0 +1,75 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _precess_cd{T<:AbstractFloat}(cd::AbstractMatrix{T}, epoch1::T, epoch2::T, crval_old::AbstractVector{T},
+                                       crval_new::AbstractVector{T}, FK4::Bool)
+    t = deg2rad((epoch2 - epoch1)*0.001)
+
+    if FK4
+        st = (epoch1 - 1900)*0.001
+        c = t*(20046.85 - st*(85.33 + st*0.37) + t*(-42.67 - st*0.37 - t*41.8))/3600
+    else
+        st = (epoch1 - 2000)*0.001
+        c = t*(20043.109 - st*(85.33 + st*0.217) + t*(-42.665 - st*0.217 - t*41.8))/3600
+    end
+    pole_ra = zero(T)
+    pole_dec = one(T)*90
+
+    if (epoch1 == 2000 && epoch2 == 1950) || (epoch1 == 1950 && epoch2 == 2000)
+        pole_ra, pole_dec = bprecess(pole_ra, pole_dec)
+    else
+        pole_ra, pole_dec = precess(pole_ra, pole_dec, epoch1, epoch2, FK4=FK4)
+    end
+    sind1, sind2 = sind.([crval_old[2], crval_new[2]])
+    cosd1, cosd2 = cosd.([crval_old[2], crval_new[2]])
+    sinra = sind(crval_new[1] - pole_ra)
+    cosfi = (cos(c) - sind1*sind2)/(cosd1*cosd2)
+    sinfi = (abs(sin(c))* sinra)/cosd1
+    r = [cosfi sinfi; -sinfi cosfi]
```

Comment:
I had transposed this `r = [cosfi -sinfi; sinfi cosfi`] because I initially wanted to keep operands sequence as r * cd, but then I changed my mind about the sequence and forgot to rectify the transposing. Corrected.